### ISSUE TITLE
Kata Containers Mission Statement

### ISSUE THREAD

**Issue by annabellebertooch** [kata-containers/community#47]
It's a good time for us to create a mission statement for the Kata Containers project! This statement should be fairly brief and tell people what this project aims to do and how we do it. 

This issue is for open discussion of what should be included in the mission statement and if anyone wants to suggest drafts.

Examples can be found from [OpenStack](https://wiki.openstack.org/wiki/Main_Page) and [Zuul](https://etherpad.openstack.org/p/zuul-mission)

**Comment by raravena80** [comment]
Suggesting something like this:

```Enable infrastructure providers to run containers in a painless, secure and isolated way via open community collaboration```

Embodies Kata Containers and the OpenStack community together.

**Comment by grahamwhaley** [comment]
Maybe we can start with a list of key items we might want to touch on? Off the top of my head, in no particular order:

- [ ] Virtual Machines
- [ ] Containers
- [ ] Small / Fast / Efficient
- [ ] Secure
- [ ] Open Source
- [ ] Open Community
- [ ] Multi-stack, neutral, independant
- [ ] Scaleable
- [ ] De-facto / reference implementation

My first stab then:

```
Define and implement the de-facto Open Source and Community implementation for
Virtual Machine Container enablement and runtimes.
```

I think we can do better though :-)

**Comment by ttx** [comment]
I think we should touch on the secure/isolation properties while being able to run classic unmodified images. To build on both suggestions above, something like:

> Openly collaborate to define and implement the de-facto open source virtual machine container solution, enabling infrastructure providers to run any container in more secure and isolated way.

**Comment by egernst** [comment]
I like it @ttx.  Only thing I would consider changing is replacing `infrastructure providers` to `users`?

**Comment by ttx** [comment]
@egernst Feel free to tear it apart and iterate on it !

I borrowed "infrastructure providers" from Ricardo's version as it's good to call those out as a primary target (in an open infrastructure story arc), but I agree it might be reductive... Maybe "...enabling infrastructure providers and other users to run..." ?

**Comment by brecky777** [comment]
I'd like to see messages around adoption / use and community. Not sure how I feel about Kata being the "defacto" because in truth, I think the road Kata has paved will likely lead to a wellspring of solutions representing a continuum of container virtualization solutions which address different use cases and levels of user sophistication.  Maybe we'd like to broaden the Kata Container umbrella to encompass more than just what we refer to a "Kata Containers" today? 
But rather than creating more questions than answers, I'll build upon what's been proposed:

Engage and nurture a robust, global community of active contributors and users to define and implement more secure and versatile container solutions that marry the benefits of virtual machines with the speed and ease-of-use of containers.

**Comment by jodh-intel** [comment]
I like it. My only comments would be to:

- Replace "marry" with "combine" (or maybe replacing "that marry" with "combining").

  Either would be clearer for non-native English speakers I think.

- Change "ease-of-use" to "ease of use"

  Although what you've written is perfectly correct, I think the doc team prefers us to drop dashes for visual clarity and again as an aid for non-native speakers.

**Comment by raravena80** [comment]
I like it too.

Engage and nurture a robust, global community of active contributors and users to define and implement more secure and versatile container solutions combining the benefits of virtual machines with the speed and ease of use of containers.

**Comment by grahamwhaley** [comment]
Personally, I'd drop the 'more' - other than that, +1 from me :-)
@annabellebertooch , how's this looking to you?

**Comment by ttx** [comment]
LGTM! I think "active" could be dropped too, as I feel like that meaning is already conveyed in "robust". If we want another adjective, I'd pick "diverse", since that has multiple desirable dimensions. As in "...a robust, diverse, and global community of contributors and users".

**Comment by annabellebertooch** [comment]
Some word-smithing for your consideration: 
"Openly collaborate across a diverse, global community of contributors and users to define and implement secure, versatile containers solutions that combine the benefits of virtual machines with the performance and ease of containers."

**Comment by annabellebertooch** [comment]
With some wordsmith assistance, I'd like to propose:
"Openly collaborate across a diverse, global community to define and implement secure, versatile containers solutions that combine the benefits of virtual machines with the performance and ease of containers”

**Comment by grahamwhaley** [comment]
Only thing I'd change is s/containers solutions/container solutions/ - but, maybe that is just a niggle happening in my head :-)

**Comment by raravena80** [comment]
Sounds good to me!

**Comment by ttx** [comment]
I like it! I'll let the choice between "container solutions" and "containers solutions" to native speakers.

**Comment by jodh-intel** [comment]
> that combine the benefits of virtual machines with the performance and ease of containers."

Since our project name includes the term "containers", this reference to the ease of containers is somewhat confusing I think. It might be better described as,

> that combine the benefits of virtual machines with the performance and ease of **Linux containers.**"

Or maybe,

> that combine the benefits of virtual machines with the performance and ease of **traditional Linux containers.**"

However, that makes what is already a very long sentence slightly longer. In fact, it's instructive to say it out loud. I timed it a few times and speaking at a "natural" rate, it takes me ~12 seconds. That suggests it's too long and complex. If so, that could prove challenging to understand for non-native English readers. 

In summary, can we break this into multiple sentences? Or -- stepping back a long way -- could we even shorten it to something super succinct like:

*Kata Containers: Working together to make containers better.* :smile:

/cc @klynnrif, @intelkevinputnam.

**Comment by gnawux** [comment]
I prefer to use **virtualization** or **virtualization technology** instead of **virtual machines**. In my opinion, we are working to leverage the capabilities of VT, and get rid of the overhead of emulating a real machine.

**Comment by annabellebertooch** [comment]
The change from "virtual machines" to "virtualization" makes sense to me. Beyond that, I'd propose we accept the below, and can always edit in the future. 

Openly collaborate across a diverse, global community to define and implement secure, versatile container solutions that combine the benefits of virtualization with the performance and ease of containers.
